eval.go

```diff
@@ -1611,8 +1611,8 @@ func (e *callExpr) eval(app *app, args []string) {
 			}
 			normal(app)
 			app.ui.cmdPrefix = "rename: "
-			extension := filepath.Ext(curr.Name())
-			if len(extension) == 0 || extension == curr.Name() || curr.IsDir() {
```

Comment:
I'm unable to figure out a need for `extension == curr.Name()` in the util function, so I skipped it there

Reply:
I think this is just to check for hidden files (e.g. `.gitignore`), which you have already covered